BetterTogetherApp/templates/BetterTogetherApp/share_promotion_create.html

```diff
@@ -1,9 +1,50 @@
-<h1>Create new SharePromotion event</h1>
-<form action='{% url "BetterTogetherApp:create_share_promotion" %}' method="post">
-    {% csrf_token %}
-    {{ form }}
-    <br><h3>Select the time to meet with the others: </h3>{{ date_time }}
-    <button type="submit">Submit</button>
-</form>  
-
-<a href="{% url 'BetterTogetherApp:share_promotion_index1' %}"> Go BACK to index</a>
\ No newline at end of file
+{% load static %}
+<html>
+
+<head>
+    <title> Create SharePromotion Event </title>
+    <link href="{% static "css/sharepromotion-create-style.css" %}" rel="stylesheet">
+    <link rel="icon" href="{% static "image/puzzle.png" %}">
+    <link href="https://fonts.googleapis.com/css?family=Hind|Lobster|Roboto&display=swap" rel="stylesheet">
```

Comment:
You should put the import font to css file

Reply:
already fixed it